src/qtism/data/storage/xml/marshalling/SimpleAssociableChoiceMarshaller.php

```diff
@@ -55,42 +59,60 @@ protected function unmarshallChildrenKnown(DOMElement $element, QtiComponentColl
                     $component->setFixed($fixed);
                 }
 
-                if (Version::compare($version, '2.1.0', '>=') === true && ($templateIdentifier = $this->getDOMElementAttributeAs($element, 'templateIdentifier')) !== null) {
+                if (Version::compare(
+                        $version,
+                        '2.1.0',
+                        '>='
+                    ) === true && ($templateIdentifier = $this->getDOMElementAttributeAs(
+                        $element,
+                        'templateIdentifier'
+                    )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?